qiskit/wrapper/defaultqiskitprovider.py

```diff
@@ -87,6 +88,12 @@ def add_provider(self, provider, provider_name):
         """
         Add a new provider to the list of known providers.
 
+        Note:
+            If a backend from the new provider has a name that is in use by an
```

Comment:
Non-blocking but what about: `If some backend of the new provider has a name in use by an already registered provider, the backend will not be available, and the name will always refer to that previously registered.`